using-mobile-extensions/adobe-media-analytics/adobe-media-release-notes.md

```diff
@@ -5,7 +5,7 @@
 ### Android Media 2.1.2
 
 * Enabled debugging post processed data in Media Analytics Assurance workflow.
-* Handled reset identities (MobileCore.resetIdentities()) event.
+* Added support to handle MobileCore.resetIdentities() API.
```

Comment:
I'd go with "... to handle the MobileCore.resetIdentities() API."

Reply:
Updated

Reply:
@calebk1, can we do the final review and merge? Thanks.